src/cascade/model/age_time_grid.py

```diff
@@ -35,8 +35,8 @@ class AgeTimeGrid:
     """
     def __init__(self, ages, times, columns):
         try:
-            self.ages = np.array(ages, dtype=np.float)
-            self.times = np.array(times, dtype=np.float)
+            self.ages = np.sort(np.atleast_1d(ages).astype(np.float))
+            self.times = np.sort(np.atleast_1d(times).astype(np.float))
```

Comment:
The atleast_1d function will turn a single age or time into a list of ages or times. This makes the user interface easier b/c you don't need to do AgeTimeGrid([50], [2000]) and can do AgeTimeGrid(50, 2000), if it's one point.